absorb/main.py

```diff
@@ -12,7 +12,6 @@
 @click.group()
 def cli() -> None:
     """Creates the main click group for absorb."""
```

Comment:
Can you please let me know that is this working while running locally?

I'm a bit skeptical on the removal of `pass` (although it should work fine)

Reply:
Yes, this works. Tested locally.
If you dont want to removed it, the line will be shwon as Missing in the coverage.

Reply:
Thanks for the heads up!